src/ts/voiceDim.ts

```diff
@@ -135,6 +135,13 @@ const otherQueries = {
   'deep site': 'is:deepsight',
   wishlist: 'is:wishlist',
   wishlisted: 'is:wishlist',
+  favorite: 'tag:favorite',
+  keeper: 'tag:keep',
+  junk: 'tag:junk',
+  infusion: 'tag:infuse',
+  archived: 'tag:archive',
+  tagged: 'is:tagged',
+  'with notes': 'is:hasnotes',
```

Comment:
`with notes` could possibly present issues with how perks are parsed out. I use `with` as a prefix/trigger for looking for perks. I think everything else is ok, though.

Reply:
ok, I will think about how to word this better